scripts/check-os.js

```diff
@@ -33,8 +33,21 @@ export default () => new Promise(async (resolve, reject) => {
         await execa('curl', ['--version'])
         log.success('"curl" found')
       } else if (info.type === 'linux') {
-        await execa('apt-get', ['--version'])
-        log.success('"apt-get" found')
+        switch (info.distro) {
```

Comment:
Given the number of different distributions, I think it would be easier to to just try the most popular package managers.

Maybe try something like this?
```js
function checkPM(name){
    await execa(name, ['--version'])
    return 'name'
}

let packageManager;
try {
    await Promise.any([
        checkPM('pacman'),
        checkPM('apt-get'),
        checkPM('dnf'),
    ])
} catch (error) {
    throw new Error('Unknown package manager')
}
log.success(`${packageManager} found`)
```

Reply:
Yeah definitely an improvement! I'll get around to fixing that, thanks. I don't suppose you had the same issue with running the test suite, what I mentioned in the PR description?